projects/components/src/time-range/time-range-for-page/time-range-for-page.component.ts

```diff
@@ -0,0 +1,37 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ActivatedRoute, UrlSegment } from '@angular/router';
+import { NavigationService, TimeRange, TimeRangeForPageService } from '@hypertrace/common';
+import { isNil } from 'lodash-es';
+
+@Component({
+  selector: 'ht-time-range-for-page',
```

Comment:
user-specified-time-range-selector

Reply:
done, as discussed, also moved to projects/observability instead of projects/components